nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/resources/OSGI-INF/imaging-service-contrib.xml

```diff
@@ -24,11 +24,15 @@
       default="true" rendition="true" />
 
     <pictureConversion id="Small" description="Small size"
-      maxSize="280" order="100" chainId="Image.Blob.Resize"
+      maxSize="560" order="100" chainId="Image.Blob.Resize"
       default="true" rendition="true" />
 
     <pictureConversion id="Medium" description="Medium size"
-      maxSize="550" order="200" chainId="Image.Blob.Resize"
+      maxSize="1000" order="200" chainId="Image.Blob.Resize"
+      default="true" rendition="true" />
```

Comment:
The commit message says "tweak" but you're now effectively making the Small size into what was previously the Medium one. That's not a tweak.